style.css

```diff
@@ -0,0 +1,78 @@
+.header {
+  display: flex;
+  color: white;
+  background-color: #3c3a39;
+  height: 10vh;
+  width: 83.8rem;
+  flex-direction: row;
+  align-items: center;
+}
+
+.name {
+  position: relative;
+  left: 40px;
+  font-family: 'Roboto', sans-serif;
+  font-size: 22px;
+  color: #fff;
+}
+
+.menu {
+  position: absolute;
+  left: 350px;
+}
+
+.main1 {
+  height: 90vh;
+  width: 83.8rem;
+  position: absolute;
+}
+
+.section1 {
+  background: #1c1a19;
+  background-image: url(./image/Illustration.png);
+  background-repeat: no-repeat;
+  background-position: 270px 300px;
```

Comment:
- You did a great job with the design and implementing changes requested by the previous reviewers. However, please make sure the webpage is mobile responsive.

## This is a mobile responsive design:
![image](https://user-images.githubusercontent.com/92215878/204843683-0da6992f-ad45-4156-b628-6a34065c3486.png)

##Your design:
![image](https://user-images.githubusercontent.com/92215878/204843810-f3f1bf57-dfdc-4b60-8fa0-34012ad0dfc5.png)